scholarx/archive/functions.js

```diff
@@ -82,21 +90,71 @@ function renderAllProfiles() {
     $("#mentorProfiles").html(mentorProfiles);
     $("#menteeProfiles").html(menteeProfiles);
 }
+function uncheckCheckboxes(){
+    for(let i=2019; i<=years[years.length-1]; i++){
```

Comment:
instead of 2019 we can get the first one from the array after it's sorted. 
```suggestion
    for(let i=years.sort()[0]; i<=years[years.length-1]; i++){
```